bundle/manifests/siteconfig.clusterserviceversion.yaml

```diff
@@ -509,8 +504,13 @@ spec:
                   initialDelaySeconds: 5
                   periodSeconds: 10
                 resources:
+                  limits:
+                    cpu: "2"
+                    ephemeral-storage: 10Gi
+                    memory: 16Gi
```

Comment:
These are quite large.
Do we have any idea what the actual limits should be?

Reply:
Initially there were no limits, which in theory implies that the pod could consume much more resources. I put these values as max values. We can revisit them once we have an analysis from the Perf team? Feel free to suggest alternative limits :)